plugins/single_plugins/switcher.cpp

```diff
@@ -781,7 +781,13 @@ class WayfireSwitcher : public wf::plugin_interface_t
         views.push_back(std::move(sv));
     }
 
-    // TODO:!!! fini
+    void fini() override
+    {
```

Comment:
```suggestion
    {
        if (output->is_plugin_active(grab_interface->name))
            deinit_switcher();
```

Reply:
This would be if the plugin is disabled while it is active. I can't imagine how this can happen during regular usage, the user will probably have to try to intentionally break it, but still ..